assets/css/less/wpbdp.less

```diff
@@ -1,1083 +1,13 @@
-        display: none;
-    }
-
-    .wpbdp-hide-on-mobile {
-        display: none;
-    }
-
-    .wpbdp-bar form.wpbdp-search-form {
-       /* width: 49%;*/
-        display: block;
-        margin-left: 0;
-        margin-top: 10px;
-    }
-    .wpbdp-bar form.wpbdp-search-form  #intextbox{
-        margin-bottom: 5px;
-        padding: 4px;
-    }
-
-    .wpbdp-bar form.wpbdp-search-form input[type="text"] {
-        padding: 4px 0;
-        margin: 0 0 2px 0;
-    }
-
-	.wpbdp-main-box .search-fields .box-row,
-	.wpbdp-main-box .main-fields.box-row form {
-		display: block;
-	}
-
-	.wpbdp-main-box .submit-btn {
-		width: 100%;
-	}
-
-	.wpbdp-main-links .button {
-	    width: 100%;
-	}
-
-    .wpbdp-listing.wpbdp-listing {
-        font-size: 90%;
-    }
-
-    .wpbdp-listing.wpbdp-listing-excerpt .field-value > label {
-        display: block;
-    }
-
-    .wpbdp-listing.wpbdp-listing-excerpt .listing-thumbnail,
-    .wpbdp-listing.wpbdp-listing-single .listing-thumbnail {
-        /*width: 40%;*/
-        float: none;
-        padding: 5px;
-    }
-
-    .wpbdp-listing.wpbdp-listing-excerpt .listing-details,
-    .wpbdp-listing.wpbdp-listing-single .listing-details {
-        margin: 0 5px;
-        float: none;
-        display: block;
-    }
-
-    .wpbdp-listing .listing-actions input {
-        font-size: 70%;
-    }
-
-    .wpbdp-listing.wpbdp-listing-single .main-image {
-        display: block;
-        float: none;
-        padding: 0;
-        margin: 0 0 10px 0;
-        text-align: center;
-        max-width: 90%;
-    }
-
-    .wpbdp-listing.wpbdp-listing-single .field-value > label {
-        display: block;
-    }
-
-    .wpbdp-submit-page.step-images #image-upload-dnd-area {
-        font-size: 90%;
-        float: none !important;
-        width: 100% !important;
-    }
-
-    .wpbdp-submit-page.step-images .dnd-area-inside-error {
-        margin-top: 30px;
-    }
-
-    .wpbdp-submit-page.step-images #image-upload-conditions {
-        width: 100% !important;
-        float: none !important;
-        font-size: 90%;
-    }
-
-    .wpbdp-submit-page.step-images #image-upload-conditions dl {
-        margin: 0;
-        padding: 0;
-    }
-
-    .wpbdp-submit-page.step-images #image-upload-conditions dl dt {
-        margin: 0;
-        margin-right: 5px;
-        padding: 0;
-        float: left;
-    }
-
-    .wpbdp-submit-page.step-images #image-upload-conditions dl dd {
-        margin: 0;
-        padding: 0;
-        display: block;
-    }
-
-    .wpbdp-submit-page.step-images .wpbdp-image img {
-        max-width: 50%;
-    }
-
-}
+/* Abstracts */
+@variables-dir: "./wpbdp/abstracts/variables";
+@import "@{variables-dir}/colors.less";
+@import "@{variables-dir}/box.less";
+@import "@{variables-dir}/typography.less";
+
+/* Mixins */
+@mixins-dir: "./wpbdp/abstracts/mixins";
+@import "@{mixins-dir}/mixins.less";
+
+/* Base */
+@base-dir: "./wpbdp/base";
+@import "@{base-dir}/base.less";
```

Comment:
Where did the rest of the wpbdp.less css go? If we remove all of this, it needs to be included somewhere else.

Reply:
@stephywells After I finish designing, I'll revisit the file and remove any new styles I added and restore old styles that we need. This way, only the original styles will remain.